vision/src/autogluon/vision/utils/pickle.py

```diff
@@ -0,0 +1,12 @@
+import io
+import pickle
+
+
+# https://github.com/pytorch/pytorch/issues/16797
+class CPU_Unpickler(pickle.Unpickler):
```

Comment:
nit: Perhaps `CPUUnpickler`? Unsure what the proper naming convention is here, but I don't usually see `_` in class names.

Reply:
The two UU's standing together feels weird, that's why I had the underscore.